src/crypto/load.py

```diff
@@ -0,0 +1,40 @@
+import os
+import json
+from typing import Dict
+from datetime import datetime
+
+from google.cloud.storage import Client
+from google.cloud.storage.bucket import Bucket
+from google.oauth2 import service_account
+
+from crypto.utils.api import Connection
+from crypto.utils.constants import LISTINGS_LATEST_URL
+
+
+class Load:
+
+    def __init__(self) -> None:
+        self._service_account = os.getenv("CLOUD_STORAGE")
+        self._credentials = service_account.Credentials.from_service_account_file(self._service_account)
```

Comment:
in that sense maybe make it an option to inject credentials or even the whole client instance?

Reply:
yeah, I think might go the route of the whole client